products/monitoring/api.yaml

```diff
@@ -1153,6 +1155,19 @@ objects:
                   A duration string, e.g. 10s.
                   Good service is defined to be the count of requests made to
                   this service that return in no more than threshold.
+            - !ruby/object:Api::Type::NestedObject
+              name: availability
+              description: |
+                Availability based SLI, dervied from count of requests made to this service that return successfully.
+              exactly_one_of:
+                - service_level_indicator.0.basic_sli.0.latency
+                - service_level_indicator.0.basic_sli.0.availability
+              properties:
+                - !ruby/object:Api::Type::Boolean
+                  name: enabled
```

Comment:
We could also consider adding a `ValidateFunc` that stops the field from being marked false.